src/SprykerEco/Zed/Adyen/Business/Hook/Saver/MakePayment/CreditCardSaver.php

```diff
@@ -51,11 +54,18 @@ protected function getPaymentStatus(?PaymentAdyenTransfer $paymentAdyenTransfer
     {
         if (
             $paymentAdyenTransfer
-            && $paymentAdyenTransfer->getResultCode() === $this->config->getOmsStatusRefused()
+            && $paymentAdyenTransfer->getResultCode() === 'refused'
```

Comment:
please move 'refused' and 'authorized' to const or find relative consts in config

Reply:
Rewrited